webpack/JobWizard/steps/Schedule/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { Title, Button, Form } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { ScheduleType } from './ScheduleType';
+import { RepeatOn } from './RepeatOn';
+import { QueryType } from './QueryType';
+import { StartEndDates } from './StartEndDates';
+import { repeatTypes } from '../../JobWizardConstants';
+
+const Schedule = () => {
+  const [repeatType, setRepeatType] = useState(repeatTypes.noRepeat);
+  const [repeatAmount, setRepeatAmount] = useState('');
+  const [starts, setStarts] = useState('');
+  const [ends, setEnds] = useState('');
```

Comment:
should we handle some basic date validation here?

Reply:
is there a Redmine for this one?

Reply:
maybe in the same Redmine? you can add `Fix values get cleared on scheduling tab when moving back/forward between steps`